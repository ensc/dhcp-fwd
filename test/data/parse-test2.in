user		bin
group		bin

chroot		/var/tmp
logfile		/var/tmp/dhcp-fwd.log

pidfile         /var/run/dhcp-fwd.pid

if eth0		true	false	false
if eth1		false	false	true
if eth2		true	true	false

name eth0	eth0-name
name eth1	X

ip eth0		192.168.42.23
ip eth2		10.0.0.2

server		ip 192.168.31.41
server		bcast eth0

ulimit stack	0x10
ulimit data	0x10M
ulimit core	0x10m
ulimit rss	0x10K
ulimit nproc	0x10k
loglevel	010
ulimit memlock	0x10M
ulimit as	010m
ulimit locks	010K
ulimit nofile	010k
