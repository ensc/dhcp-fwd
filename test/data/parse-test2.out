uid=1
gid=1
chroot_path='/var/tmp'
logfile_name='/var/tmp/dhcp-fwd.log'
pidfile_name='/var/run/dhcp-fwd.pid'
loglevel=8
ulimits={9, data={
  stack     -> (16, 16)
  data      -> (16777216, 16777216)
  core      -> (16000000, 16000000)
  rss       -> (16384, 16384)
  nproc     -> (16000, 16000)
  memlock   -> (16777216, 16777216)
  as        -> (8000000, 8000000)
  locks     -> (8192, 8192)
  nofile    -> (8000, 8000)}}
interfaces={len=3, data={
  'eth0', 'eth0-name', 1, 0, 0, 192.168.42.23
  'eth1', 'X', 0, 0, 1, 255.255.255.255
  'eth2', '', 1, 1, 0, 10.0.0.2}}
servers={len=2, data={
  UNICAST, 192.168.31.41
  BROADCAST, &eth0}}
